lib/peep.rb

```diff
@@ -0,0 +1,35 @@
+require 'pg'
+
+class Peep
+
+  # attr_reader :content, :posttime
+
+  # def initialize(content:, posttime:)
+  #   @content = content
+  #   @posttime = posttime
+  # end
```

Comment:
You could leave this part to do ORM, by adding the id as well! 
Maybe you started doing ORM but changed your mind at the end?